### ISSUE TITLE
Hi! is this package still actively maintained?

### ISSUE THREAD

**Issue by jarvisluong** [piranna/redux-offline-crud-rest#2]
I'm doing research on offline-first CRUD application and came across this, however, this package isn't seem to be actively managed? Is this working well for the author?

**Comment by piranna** [comment]
Yes, the package is being use in production :-) It didn't have updates in months because it didn't need them yet :-) Do you have any update improvement suggestion?

**Comment by jarvisluong** [comment]
Looking at the code, I'm struggling to figure out a way to show the newly created entity together with the already created. For example this situation:

- I have a post which have 4 comments (the post and the comments has already synced to the server)
- User add one more comment and the network is bad so assume it will only be created locally.

so how to I get all 5 comments together and if yes how do I identify them (which id should I use)

**Comment by piranna** [comment]
It should be done automatically since the local redux state has already the 5 comments. Do you have some code to replicate your problem?

**Comment by jarvisluong** [comment]
Hi! Sorry for delayed response, I actually had the same question to the redux-offline library maintainer in this issue: https://github.com/redux-offline/tools/issues/3
